requirements.txt

```diff
@@ -4,4 +4,9 @@ sphinx-book-theme~=1.1.3
 sphinx-copybutton~=0.5.2
 sphinx-panels~=0.4.1
 sphinx-tabs~=3.4.5
+sphinx-gallery
 myst-nb~=1.1.1
+# to run notebooks
+aiida-quantumespresso
+aiida-pseudo
+aiida-workgraph[widget]
```

Comment:
I think aiida-workgraph we do not want to pin, but I am not sure about the others